packages/manager/apps/observability/src/routes/Routes.constants.ts

```diff
@@ -7,6 +7,7 @@ export const subroutes = {
   metrics: 'metrics' as const,
   onboarding: 'onboarding' as const,
   dashboards: 'dashboards' as const,
+  delete: 'delete' as const,
```

Comment:
`delete` without any parameters is not a valid sub route
```suggestion
  delete: `delete/${URL_PARAMS.tenantId}` as const,
```